charts/xrd-vrouter/values.yaml

```
@@ -256,6 +255,19 @@ cpu: {}
   #               * physical core 1 has logical cores 2 and 3
   #               * etc...
   #hyperThreadingMode: "off"
+  #
+  # Deprecated cpu settings:
+  #
+  # Override the cpuset to use for XRd. Must not be used in conjunction with
+  # controlPlaneCpuset and dataPlaneCpuset.
+  # Deprecated. Use controlPlaneCpuset and dataPlaneCpuset instead.
+  # This must be a subset of the CPUs available for XRd to run on.
```

```suggestion
  # This must be a subset of the CPUs available for XRd to run on.
  # Deprecated. Use controlPlaneCpuset and dataPlaneCpuset instead.
```

nitpick: I'd probably swap these lines, but I will approve the PR either way.